www/js/util.js

```diff
@@ -1433,6 +1435,13 @@ function sendVideoUpdate() {
 
 /* chat */
 
+function stripImages(msg){
+    if (!USEROPTS.strip_image) {
+        return msg;
+    }
+    return msg.indexOf('<img') > -1 ? msg.replace(IMAGE_MATCH, "$1") : msg;
```

Comment:
it should be noted that the XSS seems to automatically lowercase all tags in filters, so trying to use <IMG> to get around this isn't of concern.